### ISSUE TITLE
updated german translations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#60]
```
for gtalksms 1.7.3 (as hg bundle)
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 22 Jan 2011 at 6:27

Attachments:
- [updated-german-translation.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-60/comment-0/updated-german-translation.bundle)

**Comment by GoogleCodeExporter** [comment]
```
Can you make a patch please ?
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 6:33

**Comment by GoogleCodeExporter** [comment]
```
No Problem, but whats wrong with the bundle?
```

Original comment by `fschm...@gmail.com` on 22 Jan 2011 at 7:31

Attachments:
- [updated-german-translation.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-60/comment-2/updated-german-translation.patch)

**Comment by GoogleCodeExporter** [comment]
```
oh, just noticed that the patch corrupts one name in the donors file. be 
carefull
```

Original comment by `fschm...@gmail.com` on 22 Jan 2011 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
Sometimes I can't apply bundle if I've got some modifications in progress.
In worst case I can merge manually with a patch.
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 7:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 7:40
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.3
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 8:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
